go.mod

```diff
@@ -18,6 +17,7 @@ require (
 )
 
 require (
+	github.com/golang/protobuf v1.5.3 // indirect
```

Comment:
golang.org/x/oauth2 indirectly depends on github.com/golang/protobuf

When https://github.com/golang/appengine/issues/228 is resolved, there should be no more indirect dependencies